### DISCUSSION TITLE
How to run on UI thread?

### DISCUSSION THREAD

**Discussion by ClaudiuHBann** [microsoft/react-native-windows#10561] [Q&A] [answered]
I have a module in C++ and I call a method from it and I set it's callback because it is an async one.
In the callback I change something from the view in react native.
The question is how can I run the callback on the UI thread from react native or C++ ?

**Comment by jonthysell** [comment, marked as answer]
You should be able to access the `UIDispatcher` on the `ReactContext` you saved off in your native module on init. https://microsoft.github.io/react-native-windows/docs/native-api/IReactContext#uidispatcher
